code/modules/mob/living/simple_animal/abnormality/teth/pale_horse.dm

```diff
@@ -0,0 +1,221 @@
+#define STATUS_EFFECT_MORTIS /datum/status_effect/mortis
+// Coded by Coxswain
+/mob/living/simple_animal/hostile/abnormality/pale_horse
+	name = "Pale Horse"
+	desc = "The riderless pale horse of the apocalypse."
+	icon = 'ModularTegustation/Teguicons/64x64.dmi'
+	icon_state = "palehorse"
+	icon_living = "palehorse"
+	icon_dead = "palehorse"
+	speak_emote = list("neighs")
+	threat_level = TETH_LEVEL
+	maxHealth = 800
+	health = 800
+	pixel_x = -16
+	base_pixel_x = -16
+	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1.5, WHITE_DAMAGE = 1.5, BLACK_DAMAGE = 1.5, PALE_DAMAGE = 0)
+	can_breach = TRUE
+	start_qliphoth = 4
+	work_chances = list(
+						ABNORMALITY_WORK_INSTINCT = 50,
+						ABNORMALITY_WORK_INSIGHT = 30,
+						ABNORMALITY_WORK_ATTACHMENT = 45,
+						ABNORMALITY_WORK_REPRESSION = 50
+						)
+	work_damage_amount = 5
+	work_damage_type = PALE_DAMAGE
+
+	ego_list = list(
+		/datum/ego_datum/weapon/revelation,
+		/datum/ego_datum/armor/revelation
+		)
+	gift_type =  /datum/ego_gifts/revelation
+	gift_message = "Your eyes have been opened."
+
+	//Quiploth counter
+	var/death_counter = 0
+	//teleport
+	var/can_act = TRUE
+	var/teleport_cooldown
+	var/teleport_cooldown_time = 30 SECONDS
+	//attack
+	var/pulse_range = 5 //from 5 up to 45 pale damage if caught by 9 clouds at once
+
+//work stuff
+/mob/living/simple_animal/hostile/abnormality/pale_horse/SuccessEffect(mob/living/carbon/human/user, work_type, pe)
+	datum_reference.qliphoth_change(1)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/pale_horse/Worktick(mob/living/carbon/human/user)
+	if(user.health < (user.maxHealth * 0.5))
+		return
+	else
+		user.apply_damage(4, PALE_DAMAGE, null, user.run_armor_check(null, PALE_DAMAGE))
```

Comment:
Fucking ouch, not even on bad tick just any?

Reply:
Yeah, it's intentional. It'll tick you down to half health then stop so with armor it's actually not too bad.

Reply:
It's a Teth, there aren't any pale outside of this one's and "Curfew"s armors for this level or before. Also due to this being a Teth and guaranteeing 48% of user's max hp as damage, before the potential 0-60% from work damage, I have a feeling this won't be grinded on unless they have rates to match, which I think they do?